traffic_ops/client/session.go

```diff
@@ -307,7 +307,7 @@ func NewNoAuthSession(toURL string, insecure bool, userAgent string, useCache bo
 	}, useCache)
 }
 
-// ErrUnlessOk returns nil and an error if the given Response's status code is anything but 200 OK. This includes reading the Response.Body and Closing it. Otherwise, the given response and error are returned unchanged.
+// ErrUnlessOk returns the response an error if the given Response's status code is anything but 200 OK. This includes reading the Response.Body and Closing it. Otherwise, the given response and a nil error are returned.
```

Comment:
Should be `the response and an error`
Also, shouldn't the comment say that this function returns three things?

Reply:
Thanks, fixed the typo and mentioned the remote address in 971302817.